Hymns/BrowseView.swift

```diff
@@ -2,10 +2,14 @@ import SwiftUI
 
 struct BrowseView: View {
     var body: some View {
-        NavigationView {
-            VStack {
-                Text("TODO: Browse")
-            }.navigationBarTitle("Browse")
+        VStack {
+            HStack {
+                Text("Browse").customTitle()
+                Spacer()
+            }
```

Comment:
Since we have this little HStack sequence repeated a bunch of times, what do you think about extracting that to like a view that takes in a string param (Browse, Favorites, etc)? Then we can just put like HomeScreenTitleView("Browse") here.

Reply:
Good idea, addressed in recent commit